soc/nordic/nrf53/nrf53_cpunet_mgmt.c

```diff
@@ -32,6 +55,11 @@ static void onoff_stop(struct onoff_manager *mgr, onoff_notify_fn notify)
 {
 	nrf_reset_network_force_off(NRF_RESET, true);
 
+	if (IS_ENABLED(CONFIG_SOC_NRF_GPIO_FORWARDER_FOR_NRF5340)) {
+		/* Assign pins back to app core. */
+		forward_pins(NRF_GPIO_PIN_SEL_APP);
```

Comment:
nit: `forward_pins` looks a bit weird in this context, perhaps the function should be renamed to e.g. `assign_pins`

Reply:
done